opesci/fields.py

```diff
@@ -24,13 +22,12 @@ def __new__(typ, name, **kwargs):
         return obj
 
     def __init__(self, *args, **kwargs):
-        template_dir = path.join(get_package_dir(), "templates")
-        staggered_dir = path.join(get_package_dir(), "templates/staggered")
-        self.lookup = TemplateLookup(directories=[template_dir, staggered_dir])
         super(Field, self).__init__()
```

Comment:
Can the constructor be removed as well? All it seems to do now is swallow arguments.